language/move-vm/runtime/src/debug.rs

```diff
@@ -96,14 +99,14 @@ impl DebugContext {
         }
     }
 
-    pub(crate) fn debug_loop(
+    pub(crate) fn debug_loop<L: Logger>(
         &mut self,
         function_desc: &Function,
         locals: &Locals,
         pc: u16,
         instr: &Bytecode,
         resolver: &Loader,
-        interp: &Interpreter,
+        interp: &Interpreter<L>,
```

Comment:
Nit & really no strong opinion on this, but `impl Logger` does seem a little bit more succinct and more consistent with the other function signatures in the Move VM.

Reply:
I did not know that and I like it! :)